docs/_guide/04-rendering-templates.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: Rendering templates
+slug: rendering-templates
+---
+
+
+A lit-html template expression does not cause any DOM to be created or updated. It's only a description of DOM, called a `TemplateResult`. To actually create or update DOM, you need to pass the `TemplateResult` to the `render()` function, along with a container to render to:
+
+```js
+import {html, render} from 'lit-html';
+
+const sayHi = (name) => html`<h1>Hello ${name}</h1>`;
+render(sayHi('Amy'), document.body);
+
+// subsequent renders will update the DOM
+render(sayHi('Zoe'), document.body);
+```
+
+## Render Options
+
+The `render` method also takes an `options` argument that allows you to specify:
+
+*   `eventContext`: The `this` value to use when invoking event listeners registered with the `@eventName` syntax.
```

Comment:
eventContext only effects listener functions. Listener objects with a `handleEvent` method are always called with the listener object as the `this` value.

Reply:
Fixed, thanks.